examples/hello/migrations/20240212161006_create_dbos_hello_tables.js

```diff
@@ -1,12 +1,12 @@
-import { Knex } from "knex";
+const { Knex } = require("knex");
```

Comment:
By writing our migrations in javascript, we eliminate the `ts-node` dependency

Reply:
So... we have a lot of other ts code.  That doesn't need all this stuff, so what's going on here?

Perhaps I don't see clearly.  What we publish to npm for hello - is it supposed to be source so people get good source to start with?  Or is it something to be deployed?  What we deploy ought to be minimal (no tsc, no dev dependencies, minimized js code)... but what we maintain should be the best code we can manage, and then some compilation process produces the optimized stuff.

Reply:
Like we talked about--`knex` doesn't compile Typescript code, but rather interprets it through `ts-node`, which is a monster dependency I want to eliminate. There may be a clever way to get `knex` to compile Typescript, but I wasn't able to figure it out.